packages/modules/pricing/src/services/pricing-module.ts

```diff
@@ -626,6 +636,7 @@ export default class PricingModuleService<
   }
 
   @InjectManager("baseRepository_")
+  @EmitEvents()
```

Comment:
What events would this emit?

Reply:
all the event in the message aggregatore

Reply:
I mean, what would they be, since I don't see us explicitly sending events in this method, but we are in some of the other ones. Or is there some convention around it?

Reply:
Got it yes, it makes sense 😄